user-guide/Advanced_Functionality/Databases/Indexing_database/Indexing_Database.md

```diff
@@ -30,4 +18,35 @@ While these are not recommended, we also still support setups with storage per D
 
 - [Specific GQI data sources](xref:Query_data_sources)
 
-![indexing database per DMS](~/user-guide/images/Indexing_database_per_DMS.svg)
+> [!IMPORTANT]
+> Elasticsearch is only supported up to version 6.8. As such, we recommend using OpenSearch instead.
+
```

Comment:
This is double. DataMiner supports only Elasticsearch 6.8.x, but Elasticsearch itself no longer provides support for that version. 

Perhaps it would be good to clarify this in this main page. Note that I'm okay with using the abridged phrase "Elasticsearch is only supported up to version 6.8" on all the other pages. It would get a bit big and unnecessarily cumbersome otherwise.